lib/fs.js

```diff
@@ -897,16 +897,40 @@ function symlink(target, path, type_, callback_) {
   validatePath(target, 'target');
   validatePath(path);
 
-  const flags = stringToSymlinkType(type);
   const req = new FSReqCallback();
   req.oncomplete = callback;
 
+  if (isWindows && type === null) {
+    let absoluteTarget;
+    try {
+      absoluteTarget = pathModule.resolve(path, '..', target);
```

Comment:
I'm not sure I understand this...
Cloud you leave a comment why this is necessary?